odl/contrib/datasets/ct/mayo.py

```diff
@@ -127,29 +125,34 @@ def load_projections(folder, indices=None):
     det_radius = (datasets[0].ConstantRadialDistance -
                   datasets[0].DetectorFocalCenterRadialDistance)
 
-    # Convert pitch and offset to odl defintions
-    pitch = (pixel_size[1] * shape[1] * datasets[0].SpiralPitchFactor *
-             src_radius / (src_radius + det_radius))
-    offset_along_axis = (datasets[0].DetectorFocalCenterAxialPosition -
-                         angles[0] / (2 * np.pi) * pitch)
+    # For unknown reasons, mayo does not include the tag
+    # "TableFeedPerRotation", which is what we want.
+    # Instead we manually compute the pitch
+    pitch = ((datasets[-1].DetectorFocalCenterAxialPosition -
+              datasets[0].DetectorFocalCenterAxialPosition) /
+             ((np.max(angles) - np.min(angles)) / (2 * np.pi)))
 
     # Get flying focal spot data
     offset_axial = np.array([d.SourceAxialPositionShift for d in datasets])
     offset_angular = np.array([d.SourceAngularPositionShift for d in datasets])
     offset_radial = np.array([d.SourceRadialDistanceShift for d in datasets])
 
-    angles_offset = angles - offset_angular
-    src_rad_offset = src_radius + offset_radial
-    offset_x = (np.cos(angles_offset) * (-src_rad_offset) -
-                np.cos(angles) * (-src_radius))
-    offset_y = (np.sin(angles_offset) * (-src_rad_offset) -
-                np.sin(angles) * (-src_radius))
-    offset_z = offset_axial
+    # Correct the angular sampling according to corrections
+    angles = angles - offset_angular
 
-    # TODO: WE CURRENTLY IGNORE THE OFFSETS DUE TO FLYING FOCAL SPOT
-    source_offsets = np.array([offset_x, offset_y, offset_z]).T
+    # TODO: Implement proper handling of flying focal spot
```

Comment:
How about you write `# TODO(adler-j)`? Then people looking at this know whom to ask without doing a `git blame`.